File: 151004/Eremeichik/rw/src/main/java/com/example/rw/model/dto/user/UserResponseTo.java
package com.example.rw.model.dto.user;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class UserResponseTo {
    private Long id;
    private String login;
    private String firstname;
    private String lastname;
}

File: 151004/Danilov/src/main/kotlin/com/danilovfa/model/user/UserResponseTo.kt
package com.danilovfa.model.user

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
data class UserResponseTo(
    val id: Long,
    val login: String,
    val password: String,
    @SerialName("firstname")
    val firstName: String,
    @SerialName("lastname")
    val lastName: String
)

fun User.toResponse() =
    UserResponseTo(
        id = id,
        login = login,
        password = password,
        firstName = firstName,
        lastName = lastName
    )
